aeron-archive/src/main/java/io/aeron/archive/client/ReplayMerge.java

```diff
@@ -524,6 +538,16 @@ else if (shouldStopAndRemoveReplay(position))
         return workCount;
     }
 
+    private long acquireNextAttemptLiveJoinBackoffMs()
+    {
+        if (attemptLiveJoinBackoffMs >= ATTEMPT_LIVE_JOIN_BACKOFF_MAX_MS)
+        {
+            return ATTEMPT_LIVE_JOIN_BACKOFF_MAX_MS;
+        }
```

Comment:
This check seems to redundant as the next line uses `min` function, i.e. the value `attemptLiveJoinBackoffMs` will not increase past `ATTEMPT_LIVE_JOIN_BACKOFF_MAX_MS`.
Was it added for efficiency reasons, i.e. to avoid a multiple and field assignment?